net/modemmanager/files/modemmanager.init

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2016 Aleksander Morgado <aleksander@aleksander.es>
+
+USE_PROCD=1
+START=70
+
+stop_service() {
+	# Load common utils
+	. /etc/modemmanager/modemmanager.common
+	# Set all configured interfaces as unavailable
+	mm_cleanup_interfaces
+}
+
+start_service() {
+	# Load common utils
+	. /etc/modemmanager/modemmanager.common
+
+	# Always make sure the rundir exists
+	mkdir -m 0755 -p "${MODEMMANAGER_RUNDIR}"
+
+	# Initially set all configured interfaces as unavailable
+	mm_cleanup_interfaces
+
+	# Report cached events (will wait for MM to be launched)
+	( mm_report_events_from_cache ) >/dev/null 2>&1 &
```

Comment:
humm... no, it isn't. the idea is to have the events from cache reported asynchronously by the script, in a background processs, so that the initialization isn't blocked while that is happening. Is this logic broken somehow?